xbmc/pvr/channels/PVRChannelGroups.cpp

```diff
@@ -536,6 +536,11 @@ bool CPVRChannelGroups::DeleteGroup(const CPVRChannelGroup &group)
     CLog::Log(LOGERROR, "PVR - %s - cannot delete internal group '%s'", __FUNCTION__, group.GroupName().c_str());
     return false;
   }
+  if (group.GroupID() <= 0)
+  {
+    CLog::Log(LOGDEBUG, "PVR - %s - cannot delete group '%s' invalid id '%i'", __FUNCTION__, group.GroupName().c_str(), group.GroupID());
```

Comment:
we should leave it false as we don't delete it either in the list nor in the database. or should it only indicates that no issue occurred?